app/components/avatar-image-fallback/avatar-image-fallback.component.ts

```diff
@@ -22,31 +22,39 @@ class AvatarImageFallback implements AvatarImageFallbackBindings {
         this.imageClass = this.avatarClass
             ? this.avatarClass
             : 'app-avatar-image-fallback__default';
+
         this.setCharCount();
+        this.setLetterProfileAvatar();
+    }
 
-        this.$element.find('img').bind('error', () => {
+    //sets ngLetterAvatar character count; max 2 characters
+    setCharCount() {
+        if (this.name) {
+            this.charCount = this.name.includes(' ') ? 2 : 1;
```

Comment:
use a regex here for "space characters" (includes single/multiple tabs and spaces) to make it more robust.

`/s+/.test(this.name) ? 2 : 1.
`
See here for example:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test